### ISSUE TITLE
About Dataset

### ISSUE THREAD

**Issue by Pulsar-ZZZZZ** [wsi-llava/WSI-LLaVA#13]
Dear authors,  
Thanks for your excellent work.  
I noticed that the currently provided datasets “WSI-Bench/WSI-Bench-test.jsonl” and “WSI-Bench/WSI-Bench-train.json” have been updated and are not entirely consistent with what is described in the paper.  
The paper mentions "strictly separated training and testing sets by cancer type to evaluate model generalization." However, in the new dataset, some of the pathological images used in the test set are also present in the training set. Therefore, I would like to ask if you could upload the original list of training and test datasets?
Thank you again!
